<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置扩展</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(250, 250, 250);
	border:none;
}
QGroupBox{
	color: rgb(0, 0, 0);
}
QScrollBar:vertical{
	border: none;
	width:8px;
	margin: 0 0 0 0;
	background: transparent;
}
QScrollBar::handle:vertical{
	background: rgb(200, 200, 200);
	border-radius:4px;
}
QScrollBar::handle:vertical:hover{
	background: rgb(160, 160, 160);
	border-radius:4px;
}
QScrollBar::handle:vertical:pressed{
	background: rgb(5, 160, 255);
	border-radius:4px;
}
QScrollBar::add-page,QScrollBar::sub-page:vertical{
	background: transparent;
}
QScrollBar::add-line,QScrollBar::sub-line:vertical{
	background: transparent;
}
QLineEdit{
	border:1px solid rgb(200,200,200);
	border-radius:5px;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	selection-background-color: rgb(0, 0, 0);
	selection-color: rgb(255, 255, 255);
}
#selectButton{
	color: rgb(0, 0, 0);
	border-radius:5px;
	background-color: rgb(0, 200, 250);
}
#selectButton:hover{
	color: rgb(0, 0, 0);
	border-radius:5px;
	background-color: rgb(0, 210, 255);
}
QPushButton{
	color: rgb(0, 0, 0);
	border-radius:5px;
	background-color: rgb(255, 255, 245);
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	border-radius:5px;
	background-color: rgb(255, 255, 252);
}
QListWidget{
	background:transparent;
	border:none;
}
QListWidget::item:hover{
	background-color:rgb(220,220,220);
	border-radius:5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="title">
      <string>插件信息管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1,1,1,1" columnstretch="1,2,1,2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="IDlabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>插件ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IDLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>插件名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>作者</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="authorLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>版本</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="versionLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>类别</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="typeLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>描述</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="descriptionLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>大小</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="sizeLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="logoButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="selectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>选择插件图标</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1,1">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <item row="0" column="4">
          <widget class="QPushButton" name="cancelButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>信息清空</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="findButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>插件查询</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/resource/find.png</normaloff>:/resource/find.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>插件信息保存</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="title">
      <string>自定义logo管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,1,3">
      <item row="0" column="1">
       <widget class="QPushButton" name="newButton">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>新增logo</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/resource/add.png</normaloff>:/resource/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="1" column="2">
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'楷体';&quot;&gt;TBox——logo说明：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'楷体';&quot;&gt;1.TBox自带的5个logo图片，存放在软件文件夹的image/logo目录下。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'楷体';&quot;&gt;2.用户自定义logo图片存放在软件文件夹的image/mylogo目录下，用户可自行增删。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'楷体';&quot;&gt;3.所有的logo请使用png格式的图片，否则会造成logo无法正常显示。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
